file/file.go

```diff
@@ -0,0 +1,109 @@
+// Package file defines functions to determine the file contents.
+// For example, IsFile() returns whether the path is a file or not with a bool value.
+package file
+
+import (
+	"io"
+	"os"
+	"path"
+	"runtime"
+	"strings"
+)
+
+const (
+	// Unknown : Don't use this bits
+	Unknown os.FileMode = 1 << (9 - iota)
+	// Readable : readable bits
+	Readable
+	// Writable : writable bits
+	Writable
+	// Executable : executable bits
+	Executable
+)
+
+// IsFile reports whether the path exists and is a file.
+func IsFile(path string) bool {
+	stat, err := os.Stat(path)
+	return (err == nil) && (!stat.IsDir())
+}
+
+// Exists reports whether the path exists.
+func Exists(path string) bool {
+	_, err := os.Stat(path)
+	return (err == nil)
+}
+
+// IsDir reports whether the path exists and is a directory.
+func IsDir(path string) bool {
+	stat, err := os.Stat(path)
+	return (err == nil) && (stat.IsDir())
+}
+
+// IsSymlink reports whether the path exists and is a symbolic link.
+func IsSymlink(path string) bool {
+	stat, err := os.Lstat(path)
+	if err != nil {
+		return false
+	}
+	if stat.Mode()&os.ModeSymlink == os.ModeSymlink {
+		return true
+	}
+	return false
+}
+
+// IsZero reports whether the path exists and is zero size.
+func IsZero(path string) bool {
+	stat, err := os.Stat(path)
+	return (err == nil) && (stat.Size() == 0)
+}
+
+// IsReadable reports whether the path exists and is readable.
+func IsReadable(path string) bool {
+	stat, err := os.Stat(path)
+	return (err == nil) && ((stat.Mode() & Readable) != 0)
+}
+
+// IsWritable reports whether the path exists and is writable.
+func IsWritable(path string) bool {
+	stat, err := os.Stat(path)
+	return (err == nil) && ((stat.Mode() & Writable) != 0)
+}
+
+// IsExecutable reports whether the path exists and is executable.
+func IsExecutable(path string) bool {
+	if runtime.GOOS == "windows" {
+		return strings.HasSuffix(path, ".exe")
+	}
+	stat, err := os.Stat(path)
+	return (err == nil) && ((stat.Mode() & Executable) != 0)
+}
+
+// IsHiddenFile reports whether the path exists and is included hidden file.
+func IsHiddenFile(filePath string) bool {
+	_, file := path.Split(filePath)
+	if IsFile(filePath) && strings.HasPrefix(file, ".") {
+		return true
+	}
+	return false
+}
+
+// Copy copy file to destination path
+func Copy(src string, dest string) error {
+	s, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer s.Close()
+
+	d, err := os.Create(dest)
+	if err != nil {
+		return err
+	}
+	defer d.Close()
+
+	_, err = io.Copy(d, s)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
`d.Close()` may flush the internal buffer, and the caller wants to know its failure.
So you need to check the error of `d.Close()`.

```suggestion
	return d.Close()
```

Reply:
（いや、マージさせてくれ…）

Reply:
I'm sleepy...
Thanks, thanks for your review, yeah.

Fix [8415937](https://github.com/nao1215/gorky/pull/1/commits/84159372feeb8fe0805d792e44e908d5c1632ebb)